MassRobotics-AMR-Sender/client.py

```diff
@@ -6,10 +6,14 @@
 import time
 import uuid
 import hashlib
-from datetime import datetime
+import random
+import math
+import sys
+from datetime import datetime, timezone, timedelta
 
-async def sendMessage():
-  uri = "ws://localhost:3000"
+OPERATIONAL_STATES = ["navigating", "idle", "disabled", "offline", "charging", "waiting", "loadingUnloading", "manualOveride"] 
```

Comment:
The operational states in our working document are similar, but not identical.  We split "waiting" into:

1. Waiting for human event (which probably covers your "loadingUnloading" state?)
2. Waiting for external event
3. Waiting for internal event

Reply:
Hi @zac-dydek I copied these values from `/AMR_Interop_Standard.json`, which seems to don't exactly follow what's in the doc. I can piggyback that change into this PR. What do you think would be good names ?

1. `waitingHumanEvent`
2. `waitingExternalEvent`
3. `waitingInternalEvent`
??

Thanks!

Reply:
I see, yes, it seems the .json file is slightly out-of-date.  Yes, let's make that change while we're at it.  The names you picked are fine with me!

Reply:
Thanks @zac-dydek I made the changes, could you review ?

Thanks!